src/views/Main/Main.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Main() {
+  return (
+    <div>
+      <h3>
+        Checkout everything the library has to offer! See our <a href='/books'>catalog</a> to learn more.
```

Comment:
Remember to use Link here within react-router-dom to prevent a full page reload.